test/e2e_node/runner/remote/run_remote.go

```diff
@@ -218,35 +215,36 @@ func main() {
 	gceImages := &internalImageConfig{
```

Comment:
if we can only pull one out, we don't need a map.

Reply:
I think the scope is keep changing. This is a minor refractor to clean up things. I do not  think we should rewrite existing code, Good for that to be in a separate CL.

Reply:
This one is separate from the change to `getGCEImages`.
`getGCEImages` only gets one image, but a node config can have multiple images, see https://github.com/kubernetes/kubernetes/pull/91543#issuecomment-637141822

Reply:
Which I think means that this part should stay as is.

Reply:
yup, I read it wrong here, scope changed.

Reply:
yes, alejandrox1@ is right.